src/test/java/juja/microservices/teams/service/TeamsServiceTest.java

```diff
@@ -1,7 +1,61 @@
 package juja.microservices.teams.service;
 
+import juja.microservices.teams.dao.TeamRepository;
+import juja.microservices.teams.entity.Team;
+import juja.microservices.teams.entity.TeamRequest;
+import juja.microservices.teams.exceptions.TeamUserExistsException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.inject.Inject;
+
 import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
 
+/**
+ * @author Andrii.Sidun
+ */
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(TeamService.class)
 public class TeamsServiceTest {
 
+    @Inject
+    @InjectMocks
```

Comment:
useless annotation in this case. should be @Inject + @MockBean or @Mock + @InjectMocks